src/mixins/ConsumerMixin.h

```diff
@@ -7,18 +7,19 @@
 #include <iostream>
 #include "src/AllowanceSemaphore.h"
 #include "src/Common.h"
+#include "src/ConnectionAutomaton.h"
 #include "src/NullRequestHandler.h"
 #include "src/Payload.h"
 #include "src/ReactiveStreamsCompat.h"
 #include "src/SmartPointers.h"
+#include "src/SubscriptionBase.h"
 #include "src/automata/StreamAutomatonBase.h"
 
 namespace reactivesocket {
 
 enum class StreamCompletionSignal;
 
 /// A mixin that represents a flow-control-aware consumer of data.
```

Comment:
Please update the comment.
Also can you move all of the method bodies from the .h file to the .cpp file?